src/assets/svg/filtering/does_not_contain.svg

```diff
@@ -9,7 +9,7 @@
             <g id="Icon-24px">
                 <polygon id="Shape" points="0 0 24 0 24 24 0 24"></polygon>
                 <mask id="mask-2" fill="white">
-                    <use xlink:href="#path-1"></use>
+                    <use xlink:href="#path-1" style="pointer-events: none;"></use>
```

Comment:
We we need to specify those inline styles? You're already applying this in the styles of the igx-icon?

Reply:
yes, this is correct. Will remove inline styles. Thank you.